docs/src/main/sphinx/sql/create-catalog.md

```diff
@@ -0,0 +1,44 @@
+# CREATE CATALOG
+
+## Synopsis
+
+```text
+CREATE CATALOG
+catalog_name
+USING connector_name
+[ WITH ( property_name = expression [, ...] ) ]
+```
+
+## Description
+
+Create a new catalog using the specified connector. The catalog is ready to be
```

Comment:
Do we really need to say that its ready? If anything we should say that it creates the necessary info on all nodes in the cluster and therefore might take some time.. if that is indeed the case (I mean how long it takes)

Reply:
This is correct, you can run a query using this catalog _immediately_ after this one completes, and it'll start running, just might be a bit slower.

Reply:
I know .. but do we really need to say that .. its like saying you can look at the results of a SQL query when the results are displayed..